### PR TITLE
Update Issue Template

### PR DESCRIPTION
**Pull request by orta**
Updates the issue template with some more up--to-date recommendations

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -3,12 +3,13 @@
 > what info we expect.  
 > Please remove this line and all above before submitting.
 
-> Before you start, are you using the CocoaPods beta release?
+> Before you start, are you using the CocoaPods release?
```
Comment (segiddins) [inline .github/ISSUE_TEMPLATE.md]: `latest CocoaPods release`

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -3,12 +3,13 @@
...
 > a lot has changed with Xcode 8 that is not backwards compatible.
 >
-> Xcode 8 Status: Only works with CocoaPods 1.1.0
-> Swift 3: You need to add a `.swift-version` file, see the CHANGELOG
-> China: Use https://gems.ruby-china.org/ instead of https://gems.ruby-china.org/ for the RC see #5910
+> Xcode 8 Status: Only works with CocoaPods 1.1.0+
+> Swift 3: You need to add a `.swift-version` file or the argument `--swift-version=VERSION`, see the CHANGELOG
```
Comment (segiddins) [inline .github/ISSUE_TEMPLATE.md]: only when linting or pushing to trunk

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -3,12 +3,13 @@
...
+> Not an issue about the CocoaPods command line app? Please use the right repo - https://github.com/CocoaPods
```
Comment (segiddins) [inline .github/ISSUE_TEMPLATE.md]: `Please file an issue in the appropriate repo`

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -3,12 +3,13 @@
...
+> Issues are for feature requests, and bugs. Questions should go in Stack Overflow
```
Comment (segiddins) [inline .github/ISSUE_TEMPLATE.md]: ~~,~~

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (DanToml) [inline .github/ISSUE_TEMPLATE.md]: Also Swift 3 is the default now, so this should probably be `2.3`

File: .github/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -3,12 +3,13 @@
 > what info we expect.  
 > Please remove this line and all above before submitting.
 
-> Before you start, are you using the CocoaPods beta release?
+> Before you start, are you using the CocoaPods release?
 > a lot has changed with Xcode 8 that is not backwards compatible.
 >
-> Xcode 8 Status: Only works with CocoaPods 1.1.0
-> Swift 3: You need to add a `.swift-version` file, see the CHANGELOG
-> China: Use https://gems.ruby-china.org/ instead of https://gems.ruby-china.org/ for the RC see #5910
```
Comment (DanToml) [inline .github/ISSUE_TEMPLATE.md]: I would also add something along the lines of `Using CocoaPods <= 0.39? See [link to blog post]`

**Comment by orta** [comment]
Updated with all feedback

**Pull request merged**